src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,41 @@
+import {useParams} from 'react-router-dom'
+import {useState, useEffect} from 'react'
+import axios from 'axios'
+
+function SingleArticle() {
+    let params = useParams()
+    const [article, setArticle] = useState()
+    const [isLoading, setIsLoading] = useState(true)
+
+  
+
+    useEffect(() => {
+        axios
+        .get(`https://mr-news-api.herokuapp.com/api/articles/${params.article_id}`)
+        .then((response) => {
+          setArticle(response.data.article)
+          setIsLoading(false)
+        })
+      }, [params.article_id])
```

Comment:
consider extracting these api calls to functions in an `api.js`.

✅ tidier components
✅ reusable code
✅ can switch axios/API easier if it's in one place